cs-test.kt

```diff
@@ -0,0 +1,14 @@
+class BadCode {
+    val largeList = List(1000000) { it }
+
+    // Inefficient way to calculate the sum of elements
+    var sum = 0
+    fun calculateSum() {
+        for (i in 0 until largeList.size) {
+            sum += largeList[i]
+        }
```

Comment:
This is a multi-line code suggestion
```suggestion
        sum = largeList.sum()
```